web/src/contexts/configContext.ts

```diff
@@ -29,6 +29,8 @@ import * as NexusDbaFormation from "@businessnjgovnavigator/content/fieldConfig/
 import * as NexusNameSearch from "@businessnjgovnavigator/content/fieldConfig/nexus-name-search.json";
 import * as PageNotFoundError from "@businessnjgovnavigator/content/fieldConfig/page-not-found-error.json";
 import * as Profile from "@businessnjgovnavigator/content/fieldConfig/profile.json";
+import * as RaffleBingoStep1 from "@businessnjgovnavigator/content/fieldConfig/raffle-bingo-step1.json";
+import * as RaffleBingoStep2 from "@businessnjgovnavigator/content/fieldConfig/raffle-bingo-step2.json";
```

Comment:
[boulder] I'm getting an error on these lines locally. Seems like these files may be missing?

Reply:
We also may have botched this file merge a bit yesterday. There's SO many changes.